SygicUIKit/Classes/Views/Cells/SYUIDetailTableViewCell.swift

```diff
@@ -0,0 +1,148 @@
+import UIKit
+
+
+/// Data source used by SYUIDetailTableViewCell to setup icon labels
+public struct SYUIDetailCellIconDataSource {
+    /// icon from SYUIIcon
+    public let icon: String
+    /// icon color
+    public let color: UIColor?
+    /// icon background color
+    public let backgroundColor: UIColor?
+    
+    public init(icon: String, color: UIColor?, backgroundColor: UIColor?) {
```

Comment:
Tento init tu cely nemusi byt, struktura si vytvori init so vsetkymi properties sama

Reply:
myslim ze som mal problem ze ked to explicitne nezadefinujes ako public tak to ostane internal co nemo byt...

Reply:
hmmm, aha :-)